app/views/books/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>New Book</h1>
+<%= form_for @book do |f| %>
+  <% if flash[:notice] %>
+    <p><%= flash[:notice] %></p>
+  <% end %>
+  <%= f.label :title%>
+  <%= f.text_field :title, type: :name, required: true %>
+  <br>
+  <%= f.label :year_published %>
+  <%= f.text_field :year_published, type: :number, min: -4000, max: 2019, required: true%>
+  <br>
+  <%= f.label :page_count %>
+  <%= f.text_field :page_count, type: :number, min: 1, required: true %>
+  <br>
+  <%= f.label :thumbnail %>
+  <%= f.text_field :thumbnail, type: :url %>
+  <br>
+  <%= f.label :authors %>
```

Comment:
Consider adding a message that prompts the user to enter the authors separated by commas.